assets/javascripts/parent_group_overview.js

```diff
@@ -41,6 +42,17 @@ function setupParentGroupOverviewAssets(group_id) {
         .getElementById('grouped_by_build_tab')
         .classList.remove('active', 'show', 'parent_group_overview_grouping_active');
       document.getElementById('grouped_by_group_tab').classList.add('active', 'parent_group_overview_grouping_active');
+      hiddenBuilds = document.querySelectorAll('.noBuildData');
+      if (hiddenBuilds.length > 0) {
+        hiddenBuilds.forEach(elem => {
+          console.log(elem.dataset.buildName);
+        });
+        addFlash(
```

Comment:
As I suggested I would just extend the "No builds" text, not a flash

Reply:
nevertheless we agree that doing so it doesnt solve the problem from the users perspective. Whats wrong with the flash?

Reply:
the results passed to the template are indeed limited by the filter. each template parses the same data. 

Now I read your comment repeately ... are you saying, that if we keep the notification should be more clear that the view is limited by the filter? or how should I interpret the last sentence?

Reply:
I would just keep the group title and replace/extend the "No builds" text depending on if there are no builds at all or just no builds shown due to the filtering